go-controller/pkg/ovn/controller/egress_services/egress_services_service.go

```diff
@@ -68,344 +71,60 @@ func (c *Controller) onServiceUpdate(oldObj, newObj interface{}) {
 	}
 
 	// We only care about LoadBalancer service updates that enable/disable egress service functionality
-	if !util.HasEgressSVCAnnotation(oldService) && !util.HasEgressSVCAnnotation(newService) &&
-		!util.HasEgressSVCHostAnnotation(oldService) && !util.HasEgressSVCHostAnnotation(newService) &&
-		!util.ServiceTypeHasLoadBalancer(oldService) && !util.ServiceTypeHasLoadBalancer(newService) {
+	if !util.ServiceTypeHasLoadBalancer(oldService) && !util.ServiceTypeHasLoadBalancer(newService) {
```

Comment:
so if old service has no load balancer and the new service has no load balancer skip processing. What about if the old service has load balancer and the new service has load balancer?

Reply:
then we want to queue it. what this says is that the only case we don't want to queue is when both the old and the new are not lbs, because it means we never had to do something with the service.

Reply:
why do we want to queue it? In case the load balancer IP changed values?

Reply:
yeah, or etp changed (maybe that's it)